app/assets/src/components/views/phylo_tree/PhyloTreeInputs.jsx

```diff
@@ -0,0 +1,237 @@
+import axios from "axios";
+import React from "react";
+import { Input, Checkbox, Button, Accordion, Menu } from "semantic-ui-react";
+
+class PhyloTreeInputs extends React.Component {
```

Comment:
could you add a comment summarizing what a 'PhyloTreeInputs' represents

Reply:
sure